WowPacketParser/Parsing/Parsers/QuestHandler.cs

```diff
@@ -1019,11 +1069,77 @@ public static void HandleQuestRequestItems434(Packet packet)
             }
 
             // flags, if any of these flags is 0 quest is not completable
-            packet.ReadUInt32("Unk flags 1"); // 2
-            packet.ReadUInt32("Unk flags 2"); // 4
-            packet.ReadUInt32("Unk flags 3"); // 8
-            packet.ReadUInt32("Unk flags 4"); // 16
-            packet.ReadUInt32("Unk flags 5"); // 64
+            QuestStatusFlags[] statusFlags = new QuestStatusFlags[] { QuestStatusFlags.None, QuestStatusFlags.None, QuestStatusFlags.None, QuestStatusFlags.None, QuestStatusFlags.None };
```

Comment:
**again**, we don't know what the individual values mean, because this is one of the few non-jam packets in 15595, and the client just does `if (f1 && f2 && f3 && f4 f&& f5) subcall_whatever()`

ALSO, copy-pasting this block of handling sucks. Try refactoring it

Reply:
Some values are known since there is a logical behavior when massparsing sniffs and comparing them.

But yeah thought about refactoring